web3/utils/blocks.py

```diff
@@ -1,16 +1,24 @@
 from eth_utils import (
-    force_text,
+    is_bytes,
     is_hex,
     is_integer,
     is_string,
+    is_text,
     remove_0x_prefix,
 )
 
 
 def is_predefined_block_number(value):
-    if not is_string(value):
+    if is_text(value):
+        value_text = value
+    elif is_bytes(value):
+        value_text = value.decode('latin-1')
```

Comment:
I'm assuming the use of `latin-1` is to ensure that we can in fact decode the bytes to a text value.  In that case, I think this deserves a comment.

Reply:
Right, this is speculatively decoding arbitrary byte values. I'll comment :+1: